src/main/java/oss/fosslight/controller/ProjectController.java

```diff
@@ -309,6 +309,7 @@ public String list(HttpServletRequest req, HttpServletResponse res, Model model)
 	public String edit(HttpServletRequest req, HttpServletResponse res, Model model) {
 		Project project = new Project();
 		project.setNoticeType(CoConstDef.CD_GENERAL_MODEL);
+		project.setOsType(CoConstDef.CD_OS_TYPE_LINUX);

```

Comment:
You can get the list of operation system type by using "getValues" function of oss.fosslight.common.CoCodeManager.
Then, osType can be set as the first item of the list instead of hard-coding.